go/vt/vttablet/tabletserver/query_executor_test.go

```diff
@@ -854,6 +853,16 @@ func TestQueryExecutorTableAclDualTableExempt(t *testing.T) {
 	if err != nil {
 		t.Fatalf("qre.Execute: %v, want: nil", err)
 	}
+
+	// Note: the query below does not pass unit tests due the planner adding
+	//		`limit 10001`
+	//query = "(select 0 as x from dual where 1 != 1) union (select 1 as y from dual where 1 != 1)"
+	//ctx = callerid.NewContext(context.Background(), nil, callerID)
+	//qre = newTestQueryExecutor(ctx, tsv, query, 0)
+	//_, err = qre.Execute()
+	//if err != nil {
+	//	t.Fatalf("qre.Execute: %v, want: nil", err)
+	//}
```

Comment:
Rather than adding a comment test. It would be better if you can file it as an issue for tracking.

Reply:
the test now passes...